xbmc/test/TestBasicEnvironment.cpp

```diff
@@ -61,9 +61,6 @@ void TestBasicEnvironment::SetUp()
    */
   g_advancedSettings.Initialize();
 
-  // Need to configure the network as some tests access the network member
-  g_application.SetupNetwork();
```

Comment:
should this be replaced by `CServiceBroker::GetNetwork().SetupNetwork()`?

Reply:
no.

Reply:
there are no network related tests?

Reply:
Network is brought up at service manager init level 1 which is called above.